foreman_plugin_template.gemspec

```diff
@@ -4,12 +4,12 @@ Gem::Specification.new do |s|
   s.name        = 'foreman_plugin_template'
   s.version     = ForemanPluginTemplate::VERSION
   s.license     = 'GPL-3.0'
-  s.authors     = ['TODO: Your name']
-  s.email       = ['TODO: Your email']
-  s.homepage    = 'TODO'
-  s.summary     = 'TODO: Summary of ForemanPluginTemplate.'
+  s.authors     = [' Your name']
+  s.email       = [' Your email']
+  s.homepage    = ''
+  s.summary     = ' Summary of ForemanPluginTemplate.'
```

Comment:
What is the reason for this? :thinking:

Reply:
for testing with foreman core, forgot to delete it :(